addons/account/migrations/13.0.1.1/post-migration.py

```diff
@@ -581,14 +581,24 @@ def migration_voucher_moves(env):
 
 
 def fill_account_move_reversed_entry_id(env):
+    # copy refund_invoice_id to reverse_entry_id
     openupgrade.logged_query(
         env.cr, """
         UPDATE account_move am
-        SET reversed_entry_id = am2.id
+        SET reverse_entry_id = am2.id
         FROM account_invoice ai
         JOIN account_invoice ai2 ON ai.refund_invoice_id = ai2.id
         JOIN account_move am2 ON am2.old_invoice_id = ai2.id
-        WHERE am.reversed_entry_id IS NULL AND am.old_invoice_id = ai.id"""
+        WHERE am.reverse_entry_id IS NULL AND am.old_invoice_id = ai.id"""
+    )
+    # copy reverse_entry_id to reversed_entry_id (the relation is reversed in 13.0)
+    openupgrade.logged_query(
+        env.cr, """
+        UPDATE account_move am
+        SET reversed_entry_id = am2.id
+        FROM account_move am2
+        WHERE am.reversed_entry_id IS NULL AND am2.reverse_entry_id = am.id
+        """
     )
```

Comment:
```suggestion
        """
    )
    openupgrade.lift_constraints(env.cr, "account_move", "reverse_entry_id")
```
this is needed. Or better if you add it at pre-migration, as you prefer.